pkg/dockerregistry/server/errorblobstore.go

```diff
@@ -107,7 +95,7 @@ func checkPendingCrossMountErrors(ctx context.Context, opts *storage.CreateOptio
 	if err != nil {
 		return err
 	}
-	return checkPendingErrors(context.GetLogger(ctx), ctx, namespace, name)
```

Comment:
previously, we differentiated between the context held by the startup code, and the request context. not sure if that's important, but this was preserving that